driver/normalizer/normalizer.go

```diff
@@ -61,43 +67,46 @@ var Preprocessors = []Mapping{
 	),
 	// drop extra info for other nodes (it duplicates other node fields)
 	Map(
-		Part("_", Obj{"extra": AnyNode(nil)}),
+		Part("_", Obj{"extra": Any()}),
 		Part("_", Obj{}),
 	),
-	// FIXME(bzz): make sure such comments are mapped properly
-	Map(
-		Part("_", Obj{
-			uast.KeyType: String("ImportNamespaceSpecifier"),
-			"leadingComments": Any(),
-		}),
-		Part("_", Obj{
-			uast.KeyType: String("ImportNamespaceSpecifier"),
-		}),
-	),
 }
 
 // Normalizers is the main block of normalization rules to convert native AST to semantic UAST.
 var Normalizers = []Mapping{
 	MapSemantic("Identifier", uast.Identifier{}, MapObj(
-		Obj{
-			"name": Var("name"),
+		Fields{
+			{Name: "name", Op: Var("name")},
+			//FIXME(bzz): map Flow variable types properly
```

Comment:
Not for this PR, but we can probably move those type annotations into `uast:Argument`'s `Type` field.

Reply:
Awesome, thank you for suggestion, moved to https://github.com/bblfsh/javascript-driver/issues/63